pay-id-api-spec/pay-id.v1.yml

```diff
@@ -0,0 +1,333 @@
+                $ref: '#/components/schemas/SignatureWrapper_Invoice'
+            application/xrpl-testnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper_Invoice'
+        '404':
+          description: Not Found
+        '503':
+          description: Service Unavailable
+      operationId: get-path-invoice
+      parameters:
+        - schema:
+            # Typed as a string to get around swagger-codegen issues.
+            # TODO(keefertaylor): Fix after https://github.com/swagger-api/swagger-codegen/issues/10105 is resolved
+            type: string
+          in: query
+          name: nonce
+          description: ''
+          required: true
+    post:
+      summary: post-invoice
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper_Compliance'
+            application/xrpl-mainnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper_Compliance'
+            application/xrpl-testnet+json:
+              schema:
+                $ref: '#/components/schemas/SignatureWrapper_Compliance'
+        '400':
+          description: Bad Request
+        '404':
+          description: Not Found
+        '413':
+          description: Request Entity Too Large
+        '422':
+          description: Unprocessable Entity
+        '451':
+          description: Unavailable due to legal reasons
+        '503':
+          description: Service Unavailable
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SignatureWrapper_Invoice'
+      operationId: post-path-invoice
+  '/{path}/receipt':
+    parameters:
+      - schema:
+          type: string
+        name: path
+        in: path
+        required: true
+    post:
+      summary: ''
+      operationId: post-path-receipt
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad Request
+        '404':
+          description: Not Found
+        '422':
+          description: Unprocessable Entity (WebDAV)
+        '503':
+          description: Service Unavailable
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Receipt'
+        description: ''
+components:
+  schemas:
+    CryptoAddressDetails:
+      title: CryptoAddressDetails
+      type: object
+      properties:
+        address:
+          type: string
+        tag:
+          type: string
+      required:
+        - address
+    PaymentInformation:
+      title: PaymentInformation
+      type: object
+      properties:
+        addressDetailsType:
+          type: string
+        addressDetails:
+          $ref: '#/components/schemas/CryptoAddressDetails'
+        proof_of_control_signature:
```

Comment:
Does this allow us to indicate optional fields?

Reply:
Yup, you'll see below (line 163) that there's a required field set. This field is optional because it's not in the required fields.

Reply:
got it, thanks